src/attr/_compat.py

```diff
@@ -1,15 +1,16 @@
 # SPDX-License-Identifier: MIT
 
-from __future__ import absolute_import, division, print_function
 
+import inspect
 import platform
 import sys
 import threading
 import types
 import warnings
 
+from collections.abc import Mapping, Sequence  # noqa
+
 
-PY2 = sys.version_info[0] == 2
 PYPY = platform.python_implementation() == "PyPy"
 PY36 = sys.version_info[:2] >= (3, 6)
```

Comment:
3.5 is also end of life, and tooling is also deteriorating.

‘attrs supports supported python versions’ is not that weird

Reply:
yeah as I wrote 3.5 is going out too. but baby steps :)